test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -25,9 +25,13 @@ contract FranchiserFactoryHandler is Test {
     // Handler ghost mapping to track the number of calls to each handler function
     mapping(bytes32 => CallCounts) public calls;
 
-    // Handler-updated booleans to track balance and voting power updates were correctly calculated
-    bool public balances_updated_correctly;
-    bool public voting_powers_updated_correctly;
+    // Handler ghost address set and mapping to track the balance of every address that is used in the invariant tests
+    EnumerableSet.AddressSet private accountBalanceAddresses;
+    mapping(address => uint256) private accountBalances;
+
+    // Handler ghost address set and mapping to track the voting power of every address that is used in the invariant tests
+    EnumerableSet.AddressSet private accountVotingPowerAddresses;
+    mapping(address => uint256) private accountVotingPowers;
```

Comment:
we should prefix this var with `ghost_`

Reply:
Done: 66e21be728f196ef3bb5ff940465124e7fa45d0d